eve/main.yml

```diff
@@ -1336,6 +1336,49 @@ stages:
             STEP_NAME: unit_tests_ui
       - Upload: *upload_final_status_artifact
 
+  unit_tests_crd_client_generator:
+    _metalk8s_internal_info:
+      junit_info: &_unit-test_crd_client_generator_junit_info
+        TEST_SUITE: unit test
+        TEST_NAME: crd client generator
+    worker:
+      type: kube_pod
+      path: eve/workers/pod-unit-tests/pod.yaml
+      images:
+        docker-unit-tests:
+          context: 'ui'
+          dockerfile: eve/workers/pod-unit-tests/ui/Dockerfile
+      vars:
+        name: "metalk8s-unit-tests-crd-client-generator-worker"
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          <<: *add_final_status_artifact_failed
+          env:
+            <<: *_env_final_status_artifact_failed
+            <<: *_unit-test_crd_client_generator_junit_info
+            STEP_NAME: unit_tests_crd_client_generator
+      - ShellCommand: *setup_cache
+      - ShellCommand:
+          name: Install CRD client generator dependencies
+          workdir: build/tools/crd-client-generator-js
+          command: >
+            npm ci --prefer-offline --no-audit
+          haltOnFailure: true
```

Comment:
Could be done in the worker image, to cache those dependencies as much as possible

Reply:
This is already done in the worker : https://github.com/scality/metalk8s/blob/development/2.8/eve/workers/pod-unit-tests/ui/Dockerfile#L35 but the build context was targeting `ui` instead of `tools/crd-client-generator-js` so wrong dependencies were cached.

Reply:
Is this still required then?